x/tally/keeper/filter.go

```diff
@@ -13,7 +13,9 @@ const (
 	filterTypeMAD  byte = 0x02
 )
 
+// FilterResult is the result of filtering.
 type FilterResult struct {
+	Executors    []string // list of executors
```

Comment:
If I understood the code...this list of executors is already "shuffled" or sanitize (with sort)
```suggestion
	Executors    []string // list of sorted/sanitized executors
```